test-e2e/static/index.js

```diff
@@ -0,0 +1,84 @@
+document
```

Comment:
isn't it better [for legibility purposes] to keep the listeners with the html?

Reply:
Hey, yes it is probably more legible to keep the listeners the way it was before, but it does not play well with content security policy because CSP does not allow inline scripts.

There is actually a `unsafe-inline` to suppress it, but we need to use `strict-dynamic` for AJS loading to work, but it causes CSP to ignore `unsafe-inline` [1]. 

Another option is to use `unsafe-hashes`[1], however it involves computing a hash of our code [2] and we'd have to do this for each listener.

For the above reasons, moving the listeners to its own javascript file is the best trade-off between effort and effectiveness I could come up with.

[1] https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src
[2] https://www.w3.org/TR/CSP3/#unsafe-hashes-usage